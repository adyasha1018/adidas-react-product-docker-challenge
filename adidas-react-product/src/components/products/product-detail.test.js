import React from "react";
import { shallow } from "enzyme";
import ProductDetailComponent from "./product-detail.component";

import { Provider } from "react-redux";
import configureStore from "redux-mock-store";

const mockStore = configureStore();
const store = mockStore();

const props = {};

describe("ProductDetailComponent", () => {
  const wrapper = shallow(
    <Provider store={store}>
      <ProductDetailComponent {...props} />
    </Provider>
  );
  it("should render ProductDetailComponent", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
