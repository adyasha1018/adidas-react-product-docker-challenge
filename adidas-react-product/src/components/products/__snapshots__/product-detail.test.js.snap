// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductDetailComponent should render ProductDetailComponent 1`] = `ShallowWrapper {}`;
